src/components/ProfileCard/ProfileCard.styles.ts

```diff
@@ -0,0 +1,236 @@
+import { Card, Avatar, Typography, Box } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import { greyScale } from '../../style-library/types/theme.helpers';
+import { SIZE_CONFIGS, VARIANT_CONFIGS } from './ProfileCard.constants';
+
+import type {
+  StyledProfileCardProps,
+  ProfileCardSize,
+  ProfileCardVariant,
+} from './ProfileCard.types';
+
+export const StyledCard = styled(Card, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<StyledProfileCardProps>(({ theme, $size }) => {
+  const sizeConfig = SIZE_CONFIGS[$size];
+
+  return {
+    padding: theme.spacing(sizeConfig.padding / 8),
+    borderRadius: theme.spacing(2),
+    boxShadow: theme.shadows[2],
+    transition: theme.transitions.create(['box-shadow', 'transform'], {
+      duration: theme.transitions.duration.short,
+    }),
+    '&:hover': {
+      boxShadow: theme.shadows[4],
+      transform: 'translateY(-2px)',
+    },
+    [theme.breakpoints.down('sm')]: {
+      padding: theme.spacing(2),
+    },
+    [theme.breakpoints.up('md')]: {
+      padding: theme.spacing(sizeConfig.padding / 8),
+    },
+  };
+});
+
+export const StyledProfileContainer = styled(Box, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<StyledProfileCardProps>(({ theme, $variant }) => {
+  const variantConfig = VARIANT_CONFIGS[$variant];
+
+  return {
+    display: 'flex',
+    flexDirection: variantConfig.direction,
+    alignItems: 'center',
+    gap: theme.spacing(2),
+    width: '100%',
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+      gap: theme.spacing(1.5),
+    },
+    [theme.breakpoints.up('md')]: {
+      flexDirection: variantConfig.direction,
+    },
+  };
+});
+
+export const StyledAvatar = styled(Avatar, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<{ $size: ProfileCardSize }>(({ theme, $size }) => {
+  const sizeConfig = SIZE_CONFIGS[$size];
+
+  return {
+    width: sizeConfig.avatarSize,
+    height: sizeConfig.avatarSize,
+    backgroundColor: theme.palette.primary.main,
+    fontSize: sizeConfig.avatarSize * 0.4,
+    fontWeight: theme.typography.fontWeightBold || 600,
+    border: `${theme.spacing(0.375)} solid ${theme.palette.background.paper}`,
```

Comment:
You use `theme.spacing(0.375)` for the border width. While MUI supports fractional spacing, using fractional spacings for borders is non-idiomatic and hides intent. Border widths are usually explicit (e.g. '1px') or themed token.

**Sugerencia:**
```typescript
Prefer an explicit thin border using theme token or 1px. Example:

border: `1px solid ${theme.palette.background.paper}`,

Or if you want to use spacing tokens, use an integer spacing or create a dedicated border token in the theme.
```